src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/docmosis/GenerateHearingNoticeDocumentCallbackHandler.java

```diff
@@ -76,4 +93,64 @@ private void buildDocument(CallbackParams callbackParams, CaseData.CaseDataBuild
 
         caseDataBuilder.hearingNoticeDocument(documents);
     }
+
+    private void postHearingFormWithCoverLetterLip(CallbackParams callbackParams, CaseData caseData) {
```

Comment:
I thinks its better to move the doc generation code from handler to service class . Same comment for all other handler classes.